spec/features/home_page_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Home', type: :request do
```

Comment:
It is not a request spec
```suggestion
RSpec.describe 'Home', type: :feature do
```